controllers/configmap_controller.go

```diff
@@ -477,6 +480,66 @@ func (r *ConfigMapReconciler) EnsureServicesConfigMapExists() error {
 	return r.createServicesConfigMapOnBootup()
 }
 
+// EnsureWICDRBAC ensures the WICD RBAC resources exist, creates them if not
+func (r *ConfigMapReconciler) EnsureWICDRBAC() error {
+	wicdResourceName := "windows-instance-config-daemon"
+	_, err := r.k8sclientset.RbacV1().RoleBindings(r.watchNamespace).Get(context.TODO(), wicdResourceName,
+		meta.GetOptions{})
+	if err != nil {
```

Comment:
Don't we need to ensure that the `RoleBinding` is what we expect and overwrite it if it is not?

Reply:
addressed with new changes